### ISSUE TITLE
Error 'leaks were detected'

### ISSUE THREAD

**Issue by littlegraycells** [jedireza/generator-hapi-style#36]
Creating a new scaffold and running `npm test` results in the following error

```
6 tests complete
Test duration: 88 ms
The following leaks were detected:WebAssembly
Coverage: 100.00%
Linting results: No issues

npm ERR! Test failed.  See above for more details.
```

**Comment by karpikpl** [comment]
that's due to outdated dependencies.
Node version is newer than the linter, hence the detection of WebAssembly
